src/imagetilesource.js

```diff
@@ -50,6 +50,7 @@
      * @extends OpenSeadragon.TileSource
      * @param {Object} options Options object.
      * @param {String} options.url URL of the image
+     * @param {String} options.referenceStripThumbnailUrl URL of the dedicated thumbnail image
```

Comment:
But, does it make sense to advertise (through documentation) other types when only "image" will be used?

At least that's how I understand it and what I think your comment conveys: https://github.com/openseadragon/openseadragon/issues/1209#issuecomment-306343636